### PR TITLE
Fixing exit code logging in remote executor

### PR DESCRIPTION
**Pull request by iwankgb**
Fixes issue of unusually high (eg. 842354754152) value of task exit code.

Summary of changes:
- fixed pointer handling.

Testing done:
- all existing tests should pass

### CODE REVIEW

**Comment by iwankgb** [comment]
run integration tests

File: pkg/executor/remote.go [hunk 1]
```diff
@@ -189,7 +187,7 @@ func (remote Remote) Execute(command string) (TaskHandle, error) {
 		stdoutFilePath:   stdoutFile.Name(),
 		stderrFilePath:   stderrFile.Name(),
 		host:             remote.targetHost,
-		exitCode:         exitCode,
+		exitCode:         &exitCode,
```
Comment (skonefal) [inline pkg/executor/remote.go]: Can we kill the pointer in `remoteTaskHandle`? This will help to reduce _pointer_ bugs.

Comment (iwankgb) [inline pkg/executor/remote.go]: @skonefal only if we have an idea about handling `nil` exit code value.

Comment (iwankgb) [inline pkg/executor/remote.go]: Or maybe not...

**Comment by iwankgb** [comment]
run integration tests

**Comment by iwankgb** [comment]
run integration tests

**Comment by iwankgb** [comment]
run integration tests

**Pull request merged**